changelogs/changelog.yaml

```diff
@@ -1 +1,4 @@
-releases: {}
+ancestor: null
```

Comment:
This file will also be overwritten by `antsibull-changelog`, this one might need to be reverted to work correctly, we'll see.